infrastructure/crowdin/crowdin_bot_scripts/crowdin-bot-config.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Configuration for crowdin-bot
+
+# CS Unplugged repo
+REPO="git@github.com:uccser/cs-unplugged.git"
+
+# Branch for upload of source content to crowdin
+TRANSLATION_SOURCE_BRANCH="develop"
+
+# Branch that new translations will be merged into
+TRANSLATION_TARGET_BRANCH="develop"
+
+# Branch that new metadata for in context localisation will be merged into
+IN_CONTEXT_L10N_TARGET_BRANCH="develop"
+
+# Name of crowdin config file
+CROWDIN_CONFIG_FILE="crowdin_content.yaml"
+
+# Code for in-context localisation pseudo language on Crowdin
+CROWDIN_PSEUDO_LANGUAGE="en-UD"
+
+# Code for in-context localisation pseudo language on CS Unplugged
+CSU_PSEUDO_LANGUAGE="xx_LR"
```

Comment:
Does `XX_RL` not need to be listed here?

Reply:
Negative - YY_RL is permanently symlinked to XX_LR, and these scripts do not need to be aware of it's existence